src/components/ContactsList/ContactRow.jsx

```diff
@@ -15,6 +15,24 @@ import ContactEmail from './Contacts/ContactEmail'
 const { getPrimaryCozy, getPrimaryPhone, getPrimaryEmail } = models.contact
 
 class ContactRow extends Component {
+  shouldComponentUpdate(nextProps) {
+    const { contact, breakpoints } = this.props
+
+    const hasContactBeenUpdated =
+      contact._rev !== nextProps.contact._rev ||
+      get(contact, 'cozyMetadata.updatedAt', '') !==
+        get(nextProps, 'contact.cozyMetadata.updatedAt', '')
```

Comment:
* Why is the 3rd parameter '' , and not nothing ? Is there value in this ?
* I think this could be extracted in a `hasDocBeenUpdated` function, this could be helpful to test it, and next maybe to extract it in cozy-client (maybe not now).

Reply:
> Why is the 3rd parameter '' , and not nothing ? Is there value in this ?
To return empty string if cozyMetadata.updatedAt is undefined... but will work with undefined value too

Reply:
I know what the 3rd argument of get does ;) If it works with `undefined`, let's not put the empty string. We should not have unmeaningful default values : here I wondered why the empty value of a Date field was changed to be an empty String, not null/undefined, and if you had encountered any weird bug.

Reply:
No, no bug, I'm using lodash here because `contact.cozyMetadata` might be undefined so I can't just do `cozyMetadata.updatedAt !== contact.cozyMetadata.updatedAt`, that's all